btas/generic/Khatri_Rao_Product.h

```diff
@@ -0,0 +1,20 @@
+#ifndef BTAS_KRP
+#define BTAS_KRP
+#include <btas/btas.h>
```

Comment:
You should follow existing precedent for code formatting, e.g. it is customary to add empty lines to add structure to the code. At the very minimum, when modifying existing code you must to follow the existing formatting conventions. When adding new code to an existing package you should also follow the established rules.